database/migrations/2016_09_19_143412_create_links_table.php

```diff
@@ -15,6 +15,10 @@ public function up()
         Schema::create('links', function (Blueprint $table) {
             $table->increments('id');
             $table->integer('member_id')->unsigned();
+            $table->tinyInteger('show_uk_phone');
```

Comment:
I checked https://laravel.com/docs/5.3/migrations and boolean is a valid type - why not use that? SQL doesn't actually support booleans so this is probably a wrapper, but it's better to use that for easier reading of the code and better abstraction (contrived example: if say some SQL database supports booleans, the database wrapper can use that)